var/spack/repos/builtin/packages/py-zfit-interface/package.py

```diff
@@ -0,0 +1,29 @@
+# Copyright 2013-2024 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class PyZfitInterface(PythonPackage):
+    """
+    zfit model fitting interface for HEP
+    """
+
+    homepage = "https://github.com/zfit/zfit-interface"
+    pypi = "zfit_interface/zfit_interface-0.0.3.tar.gz"
+
+    maintainers("jonas-eschle")
+    license("BSD-3-Clause", checked_by="jonas-eschle")
+
+    version("0.0.3", sha256="af7e8ed409f136187b2cd4def723504f9d619738668e963af388a79121239f74")
+
+    depends_on("python@3.9:", type=("build", "run"))
```

Comment:
3.7+ is supported. Spack only supports 3.8+ anyway, so you could actually just remove this line. 3.9+ is definitely wrong though

Reply:
Was too much ahead, you're right, it's 3.8